spec/features/filter_state_persistence_spec.rb

```diff
@@ -101,27 +101,34 @@ def and_i_click_the_vaccinated_link
   end
 
   def and_i_click_the_no_consent_response_link
-    click_on "Review no consent response"
+    click_on "1 child with no response"
   end
 
   def and_i_click_the_conflicting_consent_link
-    click_on "Review conflicting consent"
+    click_on "1 child with conflicting response"
   end
 
   def and_i_click_the_triage_needed_link
-    click_on "Review triage needed"
+    click_on "1 child requiring triage"
   end
 
   def and_i_click_the_register_attendance_link
-    click_on "Review register attendance"
+    click_on "1 child to register"
   end
 
   def and_i_click_the_ready_for_vaccinator_link
-    click_on "Review ready for vaccinator"
+    click_on sprintf "1 child for %s", @programme.name
```

Comment:
```suggestion
    click_on "1 child for #{@programme.name}"
```

It's not very typical to use `sprintf`.